### PR TITLE
Formatting different parts of template and application

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: packages/devextreme-cli/testing/dev-server.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ module.exports = class DevServer {
             metadata.items = items;
             metadata.baseTheme = `${themes[theme]}.${mode}`;
 
-            fs.writeFileSync(themeFilePath, JSON.stringify(metadata, null, '  '), 'utf8');
+            fs.writeFileSync(themeFilePath, JSON.stringify(metadata, null, '  ').concat('\n'), 'utf8');
```
Comment (nightskylark) [inline packages/devextreme-cli/testing/dev-server.js]: Why not just `+`?

Comment (egaluza) [reply packages/devextreme-cli/testing/dev-server.js]: Replace by string interpolation

File: packages/devextreme-cli/templates/react/application/src/themes/metadata.additional.json [hunk 2]
```diff
@@ -5,5 +5,8 @@
   "outputColorScheme": "additional",
   "makeSwatch": true,
   "base": true,
-  "widgets": ["treeview", "navbar"]
+  "widgets": [
```
Comment (nightskylark) [inline packages/devextreme-cli/templates/react/application/src/themes/metadata.additional.json]: Maybe it should be changed in angular and vue templates accordingly?

Comment (egaluza) [reply packages/devextreme-cli/templates/react/application/src/themes/metadata.additional.json]: Done

**Pull request merged**